packages/payment-processor/package.json

```diff
@@ -43,6 +43,7 @@
     "@requestnetwork/currency": "0.9.0",
```

Comment:
Change Request:

The description currently says:

> we store a bytes32 hash of the deserialized request ID

I think this is slightly misleading. We store the request ID which is a bytes32 hash of other data.

Reply:
Fixed